### DISCUSSION TITLE
Please clarify definition of "user" for pricing (Budibase dashboard vs. published app)

### DISCUSSION THREAD

**Discussion by chartpath** [Budibase/budibase#9180] [Help]
Hi! I'm kicking the tires on this and other low-code options to pair with Supabase for a side project that would actually be published for the general consumer public.

I'd like to use the Supabase user management system because of its deep integration with the DB access control.

Looking at the pricing model, the pricing looks totally fair for users who need to access the Budibase dashboard and contribute to app development, but there is no way it would scale for the number of consumer users who would need to log into the published app (never using the Budibase dashboard).

Could you clarify how that might work in terms of pricing? Specifically whether app users from Supabase who do not log into the Budibase dashboard would count towards the day passes?

Thanks!

**Comment by melohagan** [comment]
Hey @chartpath 

If you have users who use Supabase but never log into Budibase, then you do not have to pay for those users.

You only pay for users that log into Budibase, this includes developers who build Budibase apps, and authenticated users who use published Budibase apps.

Does that answer your question?
